flutter/lib/src/integrations/thread_info_integration.dart

```diff
@@ -0,0 +1,68 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'package:meta/meta.dart';
+
+import '../../sentry_flutter.dart';
+import '../isolate_helper.dart';
+
+/// Integration for adding thread information to spans.
+///
+/// This integration registers a lifecycle callback that adds thread/isolate
+/// information to spans when they are started.
+@internal
+class ThreadInfoIntegration implements Integration<SentryFlutterOptions> {
+  static const integrationName = 'ThreadInfoIntegration';
```

Comment:
```suggestion
  static const integrationName = 'ThreadInfo';
```

from now on this should be wtihout the integration suffix